examples/vrf/src/handlers/utils.rs

```diff
@@ -1,21 +1,14 @@
-use bytes::BufMut;
-use commonware_codec::FixedSize;
-use commonware_cryptography::bls12381::primitives::{group::Element, poly};
-use commonware_utils::{hex, Array};
+use commonware_codec::{EncodeSize, FixedSize, Write};
+use commonware_cryptography::bls12381::primitives::poly;
+use commonware_utils::Array;
 
 pub const ACK_NAMESPACE: &[u8] = b"_COMMONWARE_DKG_ACK_";
 
 /// Create a payload for acking a secret.
-pub fn payload<P: Array>(round: u64, dealer: &P, commitment: &[u8]) -> Vec<u8> {
-    let mut payload = Vec::with_capacity(u64::SIZE + P::SIZE + commitment.len());
-    payload.put_u64(round);
-    payload.extend_from_slice(dealer);
-    payload.extend_from_slice(commitment);
+pub fn payload<P: Array>(round: u64, dealer: &P, commitment: &poly::Public) -> Vec<u8> {
```

Comment:
I wonder if it is worth just creating an `Ack` object and implementing `Codec` over it?

Reply:
It mirrors the function [here](https://github.com/commonwarexyz/monorepo/blob/42f5f6cf5e764124700b4d2bd8919b84b51e7d76/consensus/src/ordered_broadcast/types.rs#L517)

Let me create an issue to consider modifying both of these functions at once

Reply:
https://github.com/commonwarexyz/monorepo/issues/774